bt/cmd/rds_gui

```diff
@@ -347,9 +341,10 @@ to_debug rds echo -e "${CYAN}T${NC}: iam_user is: ${iam_user}" >&3
 #     stdin, stdout, stderr, fd3, fd5
 # 4.) run with nohup
 
-TNL=( ${BT_LAUNCH_DIR}/rds_tunnel "${BT_CLUSTER}" )
+if [[ -z "${BT_LAUNCH_DIR}" ]]; then export BT_LAUNCH_DIR=${0%/*}; fi
+TNL=( ${BT_LAUNCH_DIR}/rds_tunnel_2 "${bt_cluster}" )
```

Comment:
BT_LAUNCH_DIR is never set in this script. Added definition and corrected some syntax